lib/terraform/client.rb

```diff
@@ -50,8 +50,8 @@ def output(name:)
     end
 
     def version
-      execute command: factory.version_command do |value|
-        return ::Terraform::Version.create value: value
+      @version ||= execute command: factory.version_command do |value|
```

Comment:
Looks like it was changed to return instead of yield since I looked at this last. Should be good now I think? https://github.com/newcontext-oss/kitchen-terraform/blob/ncs-alane-1.0.0/lib/terraform/client.rb#L50